### PR TITLE
Update docs to reflect behaviour of drone secure command

### PR DESCRIPTION
**Pull request by DCRichards**
* Added brew to list of OSX install options
* Documentation of `drone secure` command
* Use default filename `.drone.sec.yml` in example of generating `.drone.sec` file in usage

### CODE REVIEW

**Comment by DCRichards** [comment]
@gtaylor this one's been sat here a while, do let me know if it can be improved or there's a better initiative to update the docs, thanks!

File: content/devs/cli.md [hunk 1]
```diff
@@ -153,7 +162,38 @@ TODO
 
 # Secret Management
 
-TODO
+Drone lets you store secret variables in an encrypted `.drone.sec` file in the root of your repository. This is useful when your build requires sensitive information that should not be stored in plaintext in your YAML file.
+
+The `drone secure` command takes the plaintext YAML file containing your secret variables and generates the encrypted `.drone.sec` file. Details of the format of the plaintext YAML file and example usage can be found in the [secrets usage documentation](/usage/secrets).
```
Comment (tboerger) [inline content/devs/cli.md]: Linking of pages is done differently, please consult the Hugo documentation

**Comment by DCRichards** [comment]
@tboerger thanks for the feedback, I've made that update now. It's worth noting there's quite a few instances of the old style of linking about in the docs, if I have time I'll try and PR them.

**Comment by tboerger** [comment]
LGTM

**Comment by gdubicki** [comment]
Oh, this documentation looks useful! +1

**Comment by DCRichards** [comment]
@gdubicki thanks! If this is all cool from the Drone team shall we get it merged?

**Comment by gtaylor** [comment]
Secrets are a bit different in 0.5. This is unfortunately no longer usable. Apologies for letting it sit.

**Pull request closed without merging**